src/missing-types.d.ts

```diff
@@ -33,6 +33,20 @@ declare module 'url-loader!*' {
   export default value;
 }
 
+declare module 'file-loader!*' {
+  const value: string;
+  export default value;
+}
+
+declare module '*.worker.js' {
+  const value: string;
+  export default value;
+}
+
+declare module 'wasm-feature-detect' {
```

Comment:
This can be removed after updating `wasm-feature-detect`, right?

Reply:
Yes.

Reply:
Please make a release of `wasm-feature-detect` and update this :)

Reply:
Done.